tests/cases/seniors_app/case_seniors_app_account_statuses.yaml

```diff
@@ -455,3 +455,61 @@ TestSeniorAppActiveAccountStatusAndBehavior:
         - Type: ne
           Var: SECOND_ERROR_MESSAGE_VAR
           Value: "An unknown server-side error occurred while processing the command. Original error: Unable To Rotate Device"
+
+# CXTB-417 Senior app - Verify “Deactivated” Account Status Visuals
+TestSeniorAppDeactivatedAccountStatusAndBehavior:
```

Comment:
Name should be according to test case and scenario title. TestSeniorAppDeactivatedAccountStatusVisuals

Reply:
solved in this comit https://github.com/neveralone-chs-org/TestRay/pull/197/commits/8ee0c37915c677a2c5c31aa42ddbe282094580c2